src/LinuxTracing/PerfEventRecords.h

```diff
@@ -13,23 +13,23 @@ namespace orbit_linux_tracing {
 
 // This struct must be in sync with the SAMPLE_TYPE_TID_TIME_STREAMID_CPU in PerfEventOpen.h, as the
 // bits set in perf_event_attr::sample_type determine the fields this struct should have.
-struct __attribute__((__packed__)) perf_event_sample_id_tid_time_streamid_cpu {
+struct __attribute__((__packed__)) PerfEventSampleIdTidTimeStreamidCpu {
```

Comment:
As in the other file, I think that the snake_case to signal that these are views on raw data was useful.

Reply:
Discussed offline.